src/main/java/uk/gov/companieshouse/officerfiling/api/controller/ValidationStatusControllerImpl.java

```diff
@@ -44,6 +45,10 @@ public class ValidationStatusControllerImpl implements ValidationStatusControlle
     private boolean isTm01Enabled;
     @Value("${FEATURE_FLAG_ENABLE_AP01:true}")
     private boolean isAp01Enabled;
+    @Value("#{'${COUNTRY_LIST}'.split(';')}")
+    private List<String> countryList;
+    @Value("#{'${UK_COUNTRY_LIST}'.split(';')}")
```

Comment:
in https://github.com/companieshouse/chs-configs/pull/2462 I see `COUNTRY_LIST` but not `UK_COUNTRY_LIST`

Reply:
I have added UK_COUNTRY_LIST to all env files.

Reply:
I see it now, but it's missing the `export` command https://github.com/companieshouse/chs-configs/blob/develop/cidev/officer-filing-api/env#L14

Reply:
Now I know why my mum used to say Less Haste More Speed.....  export added. Cheers.